src/com/google/javascript/jscomp/PolymerClassDefinition.java

```diff
@@ -33,6 +33,8 @@
  * class.
  */
 final class PolymerClassDefinition {
+  /** Either 1 or 2 - the Polymer version of the definition */
+  final int version;
```

Comment:
Polymer 2 elements created with the `Polymer()` function are treated as Polymer 1 elements by the pass. They are only the Hybrid elements, correct? So behaviors for instance should be supported?

There is a significant difference in how the pass is going to treat the class based syntax elements. Namely I'm trying to make them fully work with renaming and obey the standard compiler rules regarding quoted properties. While this same behavior can be made to work with the 1.0 version elements, landing that change inside Google would probably be quite difficult.

I'm changing this to the enum - that makes sense. Just wanted you to be aware that the two are definitely not treated equivalently right now.